README.md

```diff
@@ -1,25 +1,10 @@
-# Kubernetes Template Project
+# Cluster API Charts
 
-The Kubernetes Template Project is a template for starting new projects in the GitHub organizations owned by Kubernetes. All Kubernetes projects, at minimum, must have the following files:
-
-- a `README.md` outlining the project goals, sponsoring sig, and community contact information
-- an `OWNERS` with the project leads listed as approvers ([docs on `OWNERS` files][owners])
-- a `CONTRIBUTING.md` outlining how to contribute to the project
-- an unmodified copy of `code-of-conduct.md` from this repo, which outlines community behavior and the consequences of breaking the code
-- a `LICENSE` which must be Apache 2.0 for code projects, or [Creative Commons 4.0] for documentation repositories, without any custom content
-- a `SECURITY_CONTACTS` with the contact points for the Product Security Team 
-  to reach out to for triaging and handling of incoming issues. They must agree to abide by the
-  [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
-  and will be removed and replaced if they violate that agreement.
+This project contains [Helm](https://helm.sh/) charts for [Cluster API](https://github.com/kubernetes-sigs/cluster-api).
```

Comment:
```suggestion
This project contains [Helm](https://helm.sh/) charts for [Cluster API](https://github.com/kubernetes-sigs/cluster-api) [infrastructure providers](https://cluster-api.sigs.k8s.io/reference/providers) to provision workload clusters.  These charts enable a natural continuation to utilize only `helm` instead of `clusterctl generate` to provision clusters after utilizing the helm chart from the [Cluster API operator install](https://cluster-api.sigs.k8s.io/user/quick-start-operator).  These helm charts are also convenient when paired with GitOps for provisioning numerous clusters.
```